sass/main.css

```diff
@@ -105,18 +117,59 @@ header .logo .logo_link {
   font-style: normal;
   line-height: 24px;
 }
+header nav .header_list {
+  display: none;
+}
 header nav .burgeur {
   align-items: flex-end;
   justify-content: center;
   margin-right: 70px;
 }
+@media screen and (max-width: 360px) {
+  header nav .burgeur {
+    margin-right: 110px;
+  }
+}
+@media screen and (max-width: 767px) {
+  header {
+    width: 450px;
+  }
+}
+@media screen and (min-width: 451px) and (max-width: 767px) {
+  header {
+    width: 450px;
+  }
+}
+@media screen and (min-width: 768px) and (max-width: 1199px) {
+  header {
+    width: 100%;
+  }
+}
```

Comment:
As suggested by the previous reviewer, kindly make sure your design is responsive to both desktop and mobile screen sizes🙏. Currently, your design breaks on screen sizes between `375px - 768px` as seen in the video below:

https://user-images.githubusercontent.com/37116322/228927315-2f7eb79f-970d-444d-8aad-57bc73f645b0.mp4

Please try to fix it so your project will be perfect.